### PR TITLE
Increases Teshari survivability

### PR DESCRIPTION
**Pull request by ggermanicus**
Makes tesh blood volume 400 from 280, halves speed bonus, and slightly decreases the bonus damage they take. They still go down very easily, but rescue seems to be a bit more possible.

### CODE REVIEW

**Comment by Cheedows** [comment]
#savethetesh

File: code/modules/species/station/teshari.dm [hunk 1]
```diff
@@ -26,17 +26,17 @@
 
 	darksight_range = 6
 	darksight_tint = DARKTINT_GOOD
-	slowdown = -0.1
+	slowdown = -0.05
```
Comment (Glloyd) [inline code/modules/species/station/teshari.dm]: make this 0.08

File: code/modules/species/station/teshari.dm [hunk 2]
```diff
@@ -26,17 +26,17 @@
...
 	total_health = 150
-	brute_mod = 1.35
-	burn_mod =  1.35
+	brute_mod = 1.2
+	burn_mod =  1.2
```
Comment (Glloyd) [inline code/modules/species/station/teshari.dm]: and these two 1.25

File: code/modules/species/station/teshari.dm (hunk 1 above)
Comment (Naxodile) [inline code/modules/species/station/teshari.dm]:
```suggestion
	slowdown = -0.08
```

File: code/modules/species/station/teshari.dm [hunk 3]
```diff
@@ -26,17 +26,17 @@
 
 	darksight_range = 6
 	darksight_tint = DARKTINT_GOOD
-	slowdown = -0.1
+	slowdown = -0.05
 	total_health = 150
-	brute_mod = 1.35
-	burn_mod =  1.35
+	brute_mod = 1.2
```
Comment (Naxodile) [inline code/modules/species/station/teshari.dm]:
```suggestion
	brute_mod = 1.25
```

File: code/modules/species/station/teshari.dm (hunk 2 above)
Comment (Naxodile) [inline code/modules/species/station/teshari.dm]:
```suggestion
	burn_mod =  1.25
```

**Pull request merged**